### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by sudocurse** [sudocurse/reading-list#2396]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: December 26, 2017 at 12:06AM<br>
<br>
<div style="font-family: arial,sans-serif; font-size: medium;">
<h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2BAV1xi"><font size="3" color="#1A0DAB">A neural approach under transfer learning for domain adaptation in land-cover classification using two-level cluster mapping</font></a></h3>
<div><font size="2" color="#006621">S Chakraborty, M Roy&nbsp;- Applied Soft Computing, 2017</font></div>
<div><font size="2" color="#222222">Abstract In this article, a domain adaptation (DA) technique using artificial neural networks<br>
based classifiers has been proposed using two-level cluster mapping technique by<br>
integrating the common data transformation and transfer learning approaches in a single<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C60D6q" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BAPuqJ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9y87J" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2BDnu5T"><font size="3" color="#1A0DAB">Exploring Models and Data for Remote Sensing Image Caption Generation</font></a>
</h3>
<div><font size="2" color="#006621">X Lu, B Wang, X Zheng, X Li&nbsp;- arXiv preprint arXiv:1712.07835, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Inspired by recent development of artificial satellite, remote sensing images have<br>
attracted extensive attention. Recently, noticeable progress has been made in scene<br>
classification and target detection. However, it is still not clear how to describe the remote<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9yGdI" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BAhM4A" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C5tckp" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2BBaSw3"><font size="3" color="#1A0DAB">A Distributed Frank-Wolfe Algorithm for Communication-Efficient Sparse Learning</font></a>
</h3>
<div><font size="2" color="#006621">FS Balcan</font></div>
<div><font size="2" color="#222222">Abstract Learning sparse combinations is a frequent theme in machine learning. In this<br>
paper, we study its associated optimization problem in the distributed setting where the<br>
elements to be combined are not centrally located but spread over a network. We address<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9G210" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BEhxFH" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9yGue" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2BAPxTr"><font size="3" color="#1A0DAB">A Semantic Loss Function for Deep Learning Under Weak Supervision</font></a>
</h3>
<div><font size="2" color="#006621">J Xu, Z Zhang, T Friedman, Y Liang, G Van den Broeck</font></div>
<div><font size="2" color="#222222">Abstract This paper develops a novel methodology for using symbolic knowledge in deep<br>
learning. We define a semantic loss function that bridges between neural output vectors and<br>
logical constraints. This loss function captures how close the neural network is to satisfying<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9yUli" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BCtOdA" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9k7GY" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2BAjZwF"><font size="3" color="#1A0DAB">Analytics-based approach to the study of learning networks in digital education settings</font></a>
</h3>
<div><font size="2" color="#006621">S Joksimovic - 2017</font></div>
<div><font size="2" color="#222222">Investigating howgroups communicate, build knowledge and expertise, reach consensus or<br>
collaboratively solve complex problems, became one of the main foci of contemporary<br>
research in learning and social sciences. Emerging models of communication and<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9G498" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BCYETx" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9trLm" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2BAhTNy"><font size="3" color="#1A0DAB">1 Past Research</font></a>
</h3>
<div><font size="2" color="#006621">G Sharon - 2017</font></div>
<div><font size="2" color="#222222">As a PhD. student, I was involved in the development of general, domain independent,<br>
search algorithms such as Enhanced-Partial Expansion A&lowast;[1, 2], Exponential deepening<br>
A&lowast;[3] and Meet in the Middle [4, 5, 6]. Yet my personal main research passion, and the focus<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C6Iwgy" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BAhMS8" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C9G2y2" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[HTML]</font> <a href="http://ift.tt/2BCtOu6"><font size="3" color="#1A0DAB">Subtype identification from heterogeneous TCGA datasets on a genomic scale by multi-view clustering with enhanced consensus</font></a>
</h3>
<div><font size="2" color="#006621">M Cai, L Li&nbsp;- BMC Medical Genomics, 2017</font></div>
<div><font size="2" color="#222222">The Cancer Genome Atlas (TCGA) has collected transcriptome, genome and epigenome<br>
information for over 20 cancers from thousands of patients. The availability of these diverse<br>
data types makes it necessary to combine these data to capture the heterogeneity of<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2C6SjTH" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2BDnwux" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2CayLy5" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2Bzyukv"><font size="3" color="#1A0DAB">A Deep Policy Inference Q-Network for Multi-Agent Systems</font></a>
</h3>
<div><font size="2" color="#006621">ZW Hong, SY Su, TY Shann, YH Chang, CY Lee&nbsp;- arXiv preprint arXiv:1712.07893, 2017</font></div>
<div><font size="2" color="#222222">Abstract: We present DPIQN, a deep policy inference Q-network that targets multi-agent<br>
systems composed of controllable agents, collaborators, and opponents that interact with<br>
each other. We focus on one challenging issue in such system<br>
<br>
via Gmail</font></div>
</div>
